### DISCUSSION TITLE
Add Basta to Pypi?

### DISCUSSION THREAD

**Discussion by mr-asher** [BASTAcode/BASTA#54] [Ideas]
Hi,

I'm a software engineer from ADACS currently working on a web application to run Basta. I'm currently looking into the best way to add Basta to the web application and was wondering if adding Basta as a package to Pypi has been considered?

My thinking is that firstly it would allow installing Basta via `pip install basta` and potentially running basta via pipx `pipx run BASTArun input_myfit.xml` without needing any other steps. 

This isn't necesary for the web app project. There's other ways to integrate Basta but I'll be available to help with this over the next few weeks if it's something that sounds useful. Adding to pypi would require a pypi account.

Kind regards,

Asher Leslie

**Comment by jakobmoss** [comment]
Hi Asher,

Thank you for your input and the work on a BASTA web app. We are very happy to collaborate. It sounds very interesting! 

We have discussed it before but never got around actually putting the code on Pypi. However, we will look into it again and promote it on our list of priorities. We hope to be able to have something ready quite soon.


Best regards on behalf of the BASTA team,
Jakob

**Reply by jakobmoss** [reply]
Added as a task in #55 .

**Comment by jakobmoss** [comment]
Hi Asher,

It should be available now following release [v1.5.0](https://github.com/BASTAcode/BASTA/releases/tag/v1.5.0) .

You can find the project here: https://pypi.org/project/BASTA/ . 

Please let us know if something does not work.


Best,
Jakob
